compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -298,6 +294,13 @@ class CompilationTests extends ParallelTesting {
     tests.foreach(_.delete())
   }
 
+  @Test def testOptimised: Unit = {
+    val outputDir = defaultOutputDir + "optimised/"
```

Comment:
`outputDir` is never used

Reply:
supposed to be used in the tree lines bellow

Reply:
Fixed